sensing/autoware_pointcloud_preprocessor/src/distortion_corrector/distortion_corrector.cpp

```diff
@@ -217,13 +217,12 @@ void DistortionCorrector<T>::undistortPointCloud(
   // If there is a point in a pointcloud that cannot be associated, record it to issue a warning
   bool is_twist_time_stamp_too_late = false;
   bool is_imu_time_stamp_too_late = false;
-  double global_point_stamp;
 
   for (; it_x != it_x.end(); ++it_x, ++it_y, ++it_z, ++it_time_stamp) {
     bool is_twist_valid = true;
     bool is_imu_valid = true;
 
-    global_point_stamp =
+    double global_point_stamp =
```

Comment:
```suggestion
    const double global_point_stamp =
```

Reply:
Thank you for pointing this out. I have fixed it.